api_client.py

```diff
@@ -524,20 +542,15 @@ def get_whoami_data(self, access_token):
             self.log("Error :: %s" % e)
             return {"error": e}
 
-    def get_partner_organization_tenants(self, whoami_response, access_token):
+    def get_partner_organization_tenants(self, whoami_response, access_token, page_num=1):
         """Get the tenants for partner and organization by calling tenant API.
         Arguments:
             whoami_response {object}: whoami data
             access_token {string}' JWT token value (default: {None})
+            page_num {int}: page number in case of more than 100 tenants (default: 1)
         Returns:
-            dict -- response containing whoami response or error
+            dict -- response containing list of whoami response or error in property "items"
```

Comment:
```suggestion
            A dictionary with a single "items" key that has as its value an array of dicts -- one item for each tenant managed by the partner or organization. If "tenant_id" is set in the configuration file, only that one tenant is returned in the array.
```

Reply:
You don't need "items" here. How about returning an array of tenant objects which will contain the configured tenant or all tenants?